packages/expo-dev-launcher/ios/ReactNative/EXDevLauncherBundleURLProviderInterceptor.swift

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+@objc
+public class EXDevLauncherBundleURLProviderInterceptor: NSObject {
+  @objc
+  public static var isInstalled: Bool = false {
+    willSet {
+      if isInstalled != newValue {
```

Comment:
nit - might have the unexpected effect of running `swizzle` if `isInstalled` is, for some reason in some future code, toggled from `true` to `false`

Reply:
I don't think so 😜  If you toggle it from `true` to `false` it will come back to the original implementation. 
Why? The answer is pretty simple:
We have two methods:
- `guessPackagerHost`      
- `EXDevLauncher_guessPackagerHost`
If we swizzle methods (`isInstalled` goes from `false` to `true`), the `guessPackagerHost` will point to our implementation and `EXDevLauncher_guessPackagerHost` to the original one.
If we swizzle one more time (`isInstalled` goes from `true` to `false`), the `guessPackagerHost` will point to the original implementation (because it gets the implementation from the current body of `EXDevLauncher_guessPackagerHost` pointer) and `EXDevLauncher_guessPackagerHost` to ours. It's a little bit confusing but should work well as long as you don't do anything crazy with threads.